### DISCUSSION TITLE
Compilation issues in L4T R36.3.0

### DISCUSSION THREAD

**Discussion by rajeshkumarwr** [OE4T/meta-tegra#1553] [Q&A] [answered]
Hi,

I have started integrating the Jetpack-6 GA version. I am using meta-tegra wip-l4t-r36.3.0 branch for the integration. There is a compilation issue when trying to locate a DTBO file. Pls find the logs below.

Since this is WIP branch, can i use it for Jetpack-6 GA integration?
Also could you pls confirm, whether the compiled JP6-GA image flashed on the board and tested OK?

_NOTE: Copying overlay tegra234-p3737-audio-codec-rt5658-40pin.dtbo to .
| cp: cannot stat '../jp6-ga/poky/build/tmp/deploy/images/jetson-agx-orin-devkit/tegra234-p3737-audio-codec-rt5658-40pin.dtbo': No such file or directory_

Thanks,
Rajesh

**Comment by madisongh** [comment, marked as answer]
As mentioned in the [README](https://github.com/OE4T/meta-tegra/blob/wip-l4t-r36.3.0/README.md), that is a work-in-progress branch where we've started integrating the R36.3.0 updates.   Thanks for reporting the problem with the overlay settings; I've pushed a correction for that.  But there's still quite a bit of work to do before the content is ready for general use, so be careful.

If you'd like to help with the update effort, that would be great.  There's a thread going in our [Matrix community](https://matrix.to/#/#OE4T_community:gitter.im) where you can follow along and/or volunteer.
